api/RTCPeerConnection.json

```diff
@@ -0,0 +1,3234 @@
+{
+  "api": {
+    "RTCPeerConnection": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection",
+        "support": {
+          "webview_android": [
+            {
+              "version_added": "56"
+            },
+            {
+              "prefix": "webkit",
+              "version_added": "51",
+              "version_removed": "56"
+            }
+          ],
+          "chrome": [
+            {
+              "version_added": "56"
+            },
+            {
+              "prefix": "webkit",
+              "version_added": "51",
+              "version_removed": "56"
+            }
+          ],
+          "chrome_android": [
+            {
+              "version_added": "56"
+            },
+            {
+              "prefix": "webkit",
+              "version_added": "51",
+              "version_removed": "56"
+            }
+          ],
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "22"
+          },
+          "firefox_android": [
+            {
+              "notes": "unprefixed",
```

Comment:
This is unnecessary, because there is no [`prefix`](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#prefix) value defined.

Reply:
ahh that one slipped through the net.. I'm on it